### DISCUSSION TITLE
we are using powershell version 4.7.2

### DISCUSSION THREAD

**Discussion by markyous** [SitecorePowerShell/Console#1261] [Q&A]
We are using powershell version 4.7.2 , and we wish to modify a one line in ScriptSession.cs 
```
// Avoid memory leak in reused threads
Runspace.DefaultRunspace = null;
```
how to get project solution 4.7.2 to build it with this modification and get the dll ?
as i can only see 4.7.0 then 5.0.0

**Comment by michaellwest** [comment]
I see what you mean. This may have been 4.7 and then a rerelease because of a packaging problem or some defects discovered. Is there something preventing you from upgrading?

**Comment by markyous** [comment]
unfortunately our production has 4.7.2 instead of 4.7 and i am afraid of upgrading because it may cause some bugs on production
